examples/foundational/07n-interruptible-google.py

```diff
@@ -53,7 +53,6 @@ async def main():
         stt = DeepgramSTTService(api_key=os.getenv("DEEPGRAM_API_KEY"))
 
         tts = GoogleTTSService(
-            credentials=os.getenv("GOOGLE_CREDENTIALS"),
```

Comment:
Don't you need either `credentials` or `credentials_path` to run this example? Should this be `credentials=os.getenv("GOOGLE_APPLICATION_CREDENTIALS")`?

Reply:
No, by default Google uses `GOOGLE_APPLICATION_CREDENTIALS`